dll/shellext/ntobjshex/ntobjfolder.h

```diff
@@ -58,55 +58,55 @@ class CNtObjectFolder :
 
     // IShellFolder
 
-    virtual HRESULT STDMETHODCALLTYPE EnumObjects(
+    STDMETHOD(EnumObjects)(
         HWND hwndOwner,
         SHCONTF grfFlags,
-        IEnumIDList **ppenumIDList);
+        IEnumIDList **ppenumIDList) override;
 
 protected:
-    virtual HRESULT STDMETHODCALLTYPE InternalBindToObject(
+    STDMETHOD(InternalBindToObject)(
         PWSTR path,
         const NtPidlEntry * info,
         LPITEMIDLIST first,
         LPCITEMIDLIST rest,
         LPITEMIDLIST fullPidl,
         LPBC pbcReserved,
-        IShellFolder** ppsfChild);
+        IShellFolder** ppsfChild) override;
 
-    virtual HRESULT STDMETHODCALLTYPE ResolveSymLink(
+    STDMETHOD(ResolveSymLink)(
         const NtPidlEntry * info,
-        LPITEMIDLIST * fullPidl);
+        LPITEMIDLIST * fullPidl) override;
 
 public:
-    virtual HRESULT STDMETHODCALLTYPE GetDefaultColumnState(
+    STDMETHOD(GetDefaultColumnState)(
         UINT iColumn,
-        SHCOLSTATEF *pcsFlags);
+        SHCOLSTATEF *pcsFlags) override;
 
-    virtual HRESULT STDMETHODCALLTYPE GetDetailsEx(
+    STDMETHOD(GetDetailsEx)(
         LPCITEMIDLIST pidl,
         const SHCOLUMNID *pscid,
-        VARIANT *pv);
+        VARIANT *pv) override;
 
-    virtual HRESULT STDMETHODCALLTYPE GetDetailsOf(
+    STDMETHOD(GetDetailsOf)(
         LPCITEMIDLIST pidl,
         UINT iColumn,
-        SHELLDETAILS *psd);
+        SHELLDETAILS *psd) override;
 
-    virtual HRESULT STDMETHODCALLTYPE MapColumnToSCID(
+    STDMETHOD(MapColumnToSCID)(
         UINT iColumn,
-        SHCOLUMNID *pscid);
+        SHCOLUMNID *pscid) override;
 
     // IPersistFolder
-    virtual HRESULT STDMETHODCALLTYPE Initialize(PCIDLIST_ABSOLUTE pidl);
+    STDMETHOD(Initialize)(PCIDLIST_ABSOLUTE pidl) override;
 
     // Internal
-    HRESULT STDMETHODCALLTYPE Initialize(PCIDLIST_ABSOLUTE pidl, PCWSTR ntPath);
```

Comment:
Does it need to become "virtual" (via the usage of STDMETHOD) ?

Reply:
This method is not virtual.

Reply:
Fixed.